site/content/index.html

```diff
@@ -228,6 +228,20 @@
 
 google.maps.event.addListener(chicagomarker, "click", function (e) { location.href="/events/2015-chicago/"});
 
+var pittsburghlocation = new google.maps.LatLng(40.4397, -79.9764);
+var pittsburghmarker = new MarkerWithLabel({
+    position: pittsburghlocation,
+    draggable: false,
+    raiseOnDrag: false,
+    map: map,
+    labelContent: "Pittsburgh<br>Aug 13 & 14",
+    labelAnchor: new google.maps.Point(29, 12),
+    labelClass: "labels",
+    labelStyle: { opacity: 1 }
+});
+
+google.maps.event.addListener(pittsburghmarker, "click", function (e) { location.href="/events/2015-pittsburgh/"});
+
 }
```

Comment:
This is good. Please also add it to https://github.com/jedi4ever/devopsdays-webby/blob/master/site/content/_future.txt and https://github.com/jedi4ever/devopsdays-webby/blob/master/site/content/_upcoming.txt - thanks.